stm32/test/test_sdi12/test_sdi12.c

```diff
@@ -0,0 +1,164 @@
+/**
```

Comment:
Header should be updated in general and a description on the setup required since it requires external hardware